lib/utils.ts

```diff
@@ -69,3 +69,56 @@ export function setPopupPath(path?: `/${string}`, cb: () => void = () => {}) {
 export function convertIPFStoHTTP(url: string) {
   return url.replace("ipfs://", "https://ipfs.io/ipfs/");
 }
+
+export function debounce<T extends (...args: any[]) => Promise<any>>(
```

Comment:
I didn't find it in the codes, can we remove it?

Reply:
24b2a49435efc12bc27495fa67c4c7b8e90f8daa

Too much vibe coding :/